tools/indexer/example/src/main.rs

```diff
@@ -257,6 +262,9 @@ async fn listen_blocks(mut stream: mpsc::Receiver<near_indexer::StreamerMessage>
 }
 
 fn main() -> Result<()> {
+    #[cfg(feature = "memory_stats")]
+    ALLOC.set_report_usage_interval(512 << 20).enable_stack_trace(true);
+
```

Comment:
we probably don't need this in an example code?

Reply:
My plan was to add those stats to `indexer-example`, so we can figure out why it's running out of memory.
Anyway, removed changes to indexer in order to reduce number of reviews required.